pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -350,3 +350,75 @@ func CreateObjects(client *client.Client, objects []runtime.Object) {
 	}
 	fmt.Println("\nApplication has been deployed to Kubernetes. You can run 'kubectl get deployment,svc' for details.")
 }
+
+func DeleteObjects(client *client.Client, name string) {
+	err := client.Services(api.NamespaceDefault).Delete(name)
+	if err != nil {
+		logrus.Fatalf("Error: '%v' while deleting service: %s", err, name)
+	}
+	logrus.Infof("Successfully deleted service: %s", name)
+
+	err = client.Deployments(api.NamespaceDefault).Delete(name, nil)
```

Comment:
You are right.

btw: look at  [`func ReaperFor(....`](https://github.com/kubernetes/kubernetes/blob/v1.3.5/pkg/kubectl/stop.go#L64) this might  useful.

Let me know if I can help you with that more.